app/controllers/concerns/reports/cost_analysis.rb

```diff
@@ -218,4 +226,76 @@ def add_detailed_table(pdf, period)
       end
     end
   end
+
+  def cost_analysis_report_csv
+    options = { col_sep: ';', force_quotes: true, encoding: 'UTF-8' }
+
+    csv_str = CSV.generate(**options) do |csv|
+      csv << cost_analysis_headers
+
+      cost_analysis_data.each do |data|
+        row = []
+
+        data.each do |column|
+          row << column.to_s.gsub(/<[^>]*>/, '')
+        end
+
+        csv << row
```

Comment:
```suggestion
        csv << data.map { |value| value.to_s.gsub /<[^>]*>/, '' }
```
Con Seattle y todo, fijate si funciona igual, debería.

Reply:
Un éxito